donghyeun/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+
+const postRouter = require("./postRouter");
+router.use("/posts", postRouter.router);
```

Comment:
- 보다 클린한 코드 형태를 위하여, 해당 require 문은 가장윗단에서 선언하여 주시고, 실제 활용은 아래에 따로 영역분리르 하여주세요!

```suggestion
const userRouter = require("./userRouter");
const postRouter = require("./postRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
```